flaskdeid/deid.py

```diff
@@ -0,0 +1,32 @@
+import json
+import logging
+
+from flask import Blueprint, render_template, request, session, abort, jsonify, Response, current_app, g
+from flaskdeid import medlpInterface
+import medlp
+import hutchner
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+bp = Blueprint('deid', __name__, url_prefix='/deid')
+
+@bp.route("/id_phi", methods=['POST'])
```

Comment:
Should we keep this consistent with the other endpoints? By which I mean, currently hutchner's endpoint is `/hutchner/annotate`, and medlp's is `/medlp/annotate`; should this combined one be something like `/deid/annotate` (or even just `/annotate`, no prefix)?

Reply:
I think that's fair. Settling on /deid/annotate